lib/request-handler.js

```diff
@@ -107,6 +107,9 @@ module.exports = function processRequest (options, request, response) {
                         if (headers.location) {
                             responseHeaders['Location'] = headers.location;
                         }
+                        if (headers['set-cookie']) {
```

Comment:
Rather than doing this way, We can have a callback in tailor [options](https://github.com/zalando/tailor#options) and whitelist some of the headers from primary fragment alone. We already have a similar way for filtering headers for the request.